extension/webapp/src/main/webapp/WEB-INF/conf/platform/application-registry-configuration.xml

```diff
@@ -161,38 +161,6 @@
               </field>
             </object>
           </object-param><object-param>
-            <name>development</name>
```

Comment:
No there is no impact as both application-registry and the gadget xml are stored in the jcr(portal-system ws)
I tested this case : install the gadget into some page then migrate by removing the app from the server:
the portlet is displayed but of course not working :) as the service was removed, so the end user will understand that the service no long exist.